flowlogs_reader/flowlogs_reader.py

```diff
@@ -364,13 +400,15 @@ def __init__(
         include_accounts=None,
         include_regions=None,
         thread_count=0,
+        check_column_count=False,
```

Comment:
I believe this should default to True, no? This would prevent anything currently relying on this from needing to instantiate the class differently to take advantage of the added functionality. I can't think of a scenario in which we wouldn't want to check the column count (although future scenarios may arise so I like the added option of a toggle), just think it should be True. Thoughts?